types/tar/index.d.ts

```diff
@@ -1,13 +1,14 @@
-// Type definitions for tar v1.0.1
+// Type definitions for tar 4.0
 // Project: https://github.com/npm/node-tar
-// Definitions by: Maxime LUCE <https://github.com/SomaticIT>
+// Definitions by: Maxime LUCE <https://github.com/SomaticIT>, Connor Peet <https://github.com/connor4312>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TODO: When/if typings for [fstream](https://github.com/npm/fstream) are written, refactor this typing to use it for the various streams.
 
 /// <reference types="node" />
 
-
 import stream = require("stream");
+import events = require("events");
```

Comment:
This doesn't seem to be used, might as well change the imports to use `import from`.